AlphaWallet/Common/Services/FirebaseReportService.swift

```diff
@@ -21,9 +21,9 @@ extension AlphaWallet {
             }
 
             let file = isRunningTests() ? R.file.googleServiceInfoTestsPlist() : R.file.googleServiceInfoPlist()
-            if let options = file.flatMap({ FirebaseOptions(contentsOfFile: $0.path) }) {
+            if let options = file.flatMap({ FIROptions(contentsOfFile: $0.path) }) {
```

Comment:
Does the code difference means we are using an older or newer version of Crashlytics?

Reply:
i am not quite sure, because we import same version, mabe it different for pods.